libs/feature/search/src/lib/state/effects.spec.ts

```diff
@@ -78,6 +78,19 @@ class FavoritesServiceMock {
 
 class EsServiceMock {
   getSearchRequestBody = jest.fn()
+  buildMoreOnAggregationPayload = jest.fn(() => ({
+    frame: 10,
+    notification: {
+      kind: 'N',
+      value: {
+        id: 'default',
+        key: 'abc',
+        payload: { abc: {} },
+        type: '[Search] Patch Results Aggregations',
+      },
+      hasValue: true,
+    },
+  }))
```

Comment:
This changes looks weird. I think you can probably just return an object like
```
{
      aggregations: {},
      size: 0,
      query: {}
}
```

And that should do the trick?

Reply:
Good point, thanks for the catch, I adapted this in a last commit 820433dfb9e4b964efff030ce28b1defbe688f6b along with fixing one import that broke when I rebased.